src/main/java/org/mvel2/execution/ExecutionArrayList.java

```diff
@@ -98,7 +135,19 @@ public E set(int index, E element) {
         return oldValue;
     }
 
-    public ExecutionArrayList<E> slice(int start, int end) {
+    public ExecutionArrayList<E> slice() {
+        return slice(0, this.size());
```

Comment:
return new ExecutionArrayList<>(this, this.executionContext);

Reply:
Ok